### DISCUSSION TITLE
How to add custom colors?

### DISCUSSION THREAD

**Discussion by tesar-tech** [Megabit/Blazorise#3123] [Q&A]
Front end is not my biggest friend so these questions may sound stupid..

I am using Blazorise with bootstrap, I set up everything according to [docs](https://blazorise.com/docs/start). 
Now I would like to use different color than `primary`, `danger`, etc.. Lets call it `third`. I know it is probably not possible to add new `ThemeColorOptions`. And I am not asking about theming existing color options (like change color of `danger`).

I would like to at least use my custom div with `third` color background like: 

```html
<div class="bg-third">Helou</div>
```
Configurating new colors in bootstrap is somehow done with sass files (I guess)... As I said, front end is not my cup of tea, so I don't know what to do now. Where to add sass files to not disturb current (balzorise/bootsrap) structure, etc. How to use them properly inside Blazor WA...

Thanks for any advice.

**Comment by stsrki** [comment]
Hi, adding new strongly-typed colors is not possible.

What you can do is to define a custom class name to a component and assign it to a `Class` parameter.

`<Div Class="bg-third" />`
